internal/pipeline/processor.go

```diff
@@ -167,6 +196,19 @@ func (p *Processor) processEventMatch(ctx context.Context, match EventMatch) err
 		// Continue even if status recording fails
 	}
 
+	// Add alert to SRE server if available
```

Comment:
At a high level, why do we have a callout to the server from here, rather than the server sharing the same datasource as this component?